### PR TITLE
[SVTPlay] Add subtitle support

### PR DESCRIPTION
**Pull request by maix**
basic subtitle support for svtplay.se

### CODE REVIEW

File: youtube_dl/extractor/svt.py [hunk 1]
```diff
@@ -37,6 +37,12 @@ def _extract_video(self, url, video_id):
                 })
         self._sort_formats(formats)
 
+        # SVT does not tell us the language, so we assume swedish.
+        subtitles = {}
+        for sr in video_info['subtitleReferences']:
+            if 'url' in sr:
+                subtitles.setdefault('sv', []).append({'url': sr['url']})
```
Comment (dstftw) [inline youtube_dl/extractor/svt.py]: Valid ISO 3166-1 code for Sweden is `se`.

**Comment by dstftw** [comment]
Can you add a test?

File: youtube_dl/extractor/svt.py (hunk 1 above)
Comment (maix) [inline youtube_dl/extractor/svt.py]: yes but the language code (ISO-639) for swedish is sv :) https://en.wikipedia.org/wiki/Swedish_language

**Comment by maix** [comment]
> Can you add a test?
It seems that all the videos on the site are only accessible within a specific timeframe, mostly only 7 or 30 days, a few for a year—that would probably mean that the test will fail after some time. So I'd say it's better not to include one at all, what do you think?

**Pull request closed without merging**